core/src/Kokkos_CopyViews.hpp

```
@@ -2464,7 +2520,35 @@ impl_resize(const Impl::ViewCtorProp<ViewCtorArgs...>& arg_prop,
     auto prop_copy = Impl::with_properties_if_unset(
         arg_prop, typename view_type::execution_space{}, v.label());
 
-    view_type v_resized(prop_copy, n0, n1, n2, n3, n4, n5, n6, n7);
+    view_type v_resized;
+    if constexpr (!view_type::traits::impl_is_customized) {
+      // FIXME: this should be forbidden anyway
+      v_resized = view_type(prop_copy, n0, n1, n2, n3, n4, n5, n6, n7);
+    } else {
+      // FIXME SACADO: this is specializing for sacado, might need a better
+      // thing
+      Kokkos::Impl::AccessorArg_t acc_arg{new_extents[view_type::rank()]};
+      auto prop_copy2 = Impl::with_properties_if_unset(prop_copy, acc_arg);
+      if constexpr (view_type::rank() == 0) {
```

aside from `acc_arg` why are we initializing view_type in a different way here compared to the if branch above? (i.e. initializing differently depending on rank)

---

because passing extra arguments in the View ctor actually screws up my follow up logic for "if you have exactly one extra argument then you are a Sacado construction call where I will reinterpret the extra argument as a thing I need to pass to the accessor" Normally extra arguments are just silently dropped. I separated those code paths in follow up stuff. Also in general: we should just outright forbid passing extra arguments into our ctor.